db/install.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<XMLDB PATH="admin/tool/messenger/db" VERSION="2021060401" COMMENT="XMLDB file for Moodle tool/messenger"
+       xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+       xsi:noNamespaceSchemaLocation="../../../../lib/xmldb/xmldb.xsd"
+>
+    <TABLES>
+        <TABLE NAME="tool_messenger_messagejobs" COMMENT="Table containing all messagejobs">
+            <FIELDS>
+                <FIELD NAME="id" TYPE="int" LENGTH="10" NOTNULL="true" SEQUENCE="true" COMMENT="id of the request"/>
+                <FIELD NAME="usermodified" TYPE="int" LENGTH="10" NOTNULL="true" SEQUENCE="false"/>
+                <FIELD NAME="timecreated" TYPE="int" LENGTH="10" NOTNULL="true" SEQUENCE="false"/>
+                <FIELD NAME="timemodified" TYPE="int" LENGTH="10" NOTNULL="true" SEQUENCE="false"/>
+                <FIELD NAME="message" TYPE="text" NOTNULL="true" SEQUENCE="false" COMMENT="message"/>
+                <FIELD NAME="subject" TYPE="text" NOTNULL="true" SEQUENCE="false" COMMENT="message"/>
+                <FIELD NAME="progress" TYPE="int" LENGTH="5" NOTNULL="true" SEQUENCE="false" COMMENT="current progress"/>
```

Comment:
This refers to the userid of the last processed user right? Then this should better have a length of 10 as most users in the Learnweb have at least 5-digit ids.